task/options/options.go

```diff
@@ -80,6 +192,42 @@ const (
 	optRetry       = "retry"
 )
 
+// contains is a helper function to see if an array of strings contains a string
+func contains(s []string, e string) bool {
+	for i := range s {
+		if s[i] == e {
+			return true
+		}
+	}
+	return false
+}
+
+func grabTaskOptionAST(p *ast.Package, keys ...string) map[string]ast.Expression {
+	res := make(map[string]ast.Expression, 2) // we preallocate two keys for the map, as that is how many we will use at maximum (offset and every)
+	for i := range p.Files {
+		for j := range p.Files[i].Body {
+			if p.Files[i].Body[j].Type() == "OptionStatement" {
```

Comment:
This would probably be a little more readable with inverted conditions and continues, for reduced indentation, like

```
for ... {
  if p.Files[i].Body[j].Type() != "OptionStatement" {
    continue
  }
  opt := (p.Files[i].Body[j]).(*ast.OptionStatement)
  if opt.Assignment.Type() != "VariableAssignment" {
    continue
  }
  asmt := opt.Assignment.(*ast.VariableAssignment)
  // ...
}